app/controllers/lines_controller.rb

```diff
@@ -7,8 +7,15 @@ def index
   end
 
   def show
+    authorize(@line)
     @line = Line.find(params[:id])
     @tips = @line.tips
-    authorize(@line)
+  end
+
+  def suggested_lines
+    skip_authorization # pundit to be completed
+    @area = Area.find(params[:area_id])
+    @lines = @area.lines.where(grade: 5.10..5.12)
```

Comment:
Will this range include the letters, like 5.10a, 5.11b?

Reply:
yeah

Reply:
I don't think it does...